core/src/OpenACC/Kokkos_OpenACC_DeepCopy.hpp

```diff
@@ -88,8 +88,7 @@ struct Kokkos::Impl::DeepCopy<Kokkos::Experimental::OpenACCSpace,
         "Kokkos::Impl::DeepCopy<OpenACCSpace, OpenACCSpace, "
         "ExecutionSpace>::DeepCopy: fence before copy");
     if (n > 0) {
-      acc_memcpy_device_async(dst, const_cast<void*>(src), n,
-                              exec.acc_async_queue());
+      acc_memcpy_device(dst, const_cast<void*>(src), n);
```

Comment:
Why did you change?

Reply:
This case is for non-OpenACC execution space (`exec`) to execute the memory copy (`acc_memcpy_device`), where the `exec` does not have `acc_async_queue` method, and thus synchronous memory copy should happen.
One possible alternative implementation is to assign a default async queue for the OpenACC backend, and this default async queue is used for non-OpenACC execution space to perform asynchronous memory transfer.

Reply:
Oh right